build/yaml/templates/dotnet-package-steps.yml

```diff
@@ -0,0 +1,42 @@
+steps:
+- task: UseDotNet@2
+  displayName: 'Use .Net Core sdk 3.1.x'
+  inputs:
+    version: 3.1.x
+
+- task: DotNetCoreCLI@2
+  displayName: 'Run `dotnet restore`'
+  inputs:
+    command: restore
+    projects: '$(WorkingDirectory)\*.csproj'
+    feedsToUse: config
+    nugetConfigPath: packages\NuGet.Config
+
+- task: DotNetCoreCLI@2
+  displayName: 'Run `dotnet build`'
+  inputs:
+    projects: '$(WorkingDirectory)\*.csproj'
```

Comment:
Should this not have command: 'build' somewhere?

Reply:
The default command is uses is build but I'll add it in to be more explicit